src/useGet.test.tsx

```diff
@@ -645,6 +645,63 @@ describe("useGet hook", () => {
     });
   });
 
+  describe("originalResponse", () => {
+    it("should return original response when flag is enabled", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/")
+        .reply(200, { oh: "my god 😍" });
+
+      const MyAwesomeComponent = () => {
+        const { loading, response } = useGet({
+          path: "/",
+          originalResponse: true,
+        });
+
+        return loading ? (
+          <div data-testid="loading">Loading…</div>
+        ) : (
+          <div data-testid="response">{response ? JSON.stringify(response) : null}</div>
+        );
+      };
+
+      const { getByTestId } = render(
+        <RestfulProvider base="https://my-awesome-api.fake">
+          <MyAwesomeComponent />
+        </RestfulProvider>,
+      );
+
+      await waitForElement(() => getByTestId("response"));
+      expect(getByTestId("response")).not.toBeEmpty();
```

Comment:
Could be nice to add some header or something to make sure everything is working as expected